.ci/.jenkins_nodejs.yml

```diff
@@ -0,0 +1,7 @@
+NODEJS_VERSION:
+  - "12"
+  - "11"
+  - "10"
+  - "8"
+  - "8.1"
```

Comment:
@watson We don't need 8.1 either, do we? We just have that in the agent for async_hooks vs not, which isn't used in the http client at all, right?

Reply:
Nope. As I just mentioned in https://github.com/elastic/apm-nodejs-http-client/pull/65#discussion_r301010991, then I think we either should have all or none. So if we want `8.1`, then we should also include `6.0`, `10.0`, `11.0` and `12.0` - or we should just remove `8.1`.

Reply:
I'll remove `8.1` then